NEWS.md

```diff
@@ -168,6 +168,7 @@ function calls. (#850, #851, @renkun-ken)
 * Several linters tightened internal logic to allow for raw strings like `R"( a\string )"` (#1034, @michaelchirico)
 * `object_usage_linter()` correctly detects functions assigned with `=` instead of `<-` (#1081, @michaelchirico)
 * `undesirable_function_linter()` no longer lints undesirable symbols if they are used as names (#1050, @AshesITR)
+* Moved the default lintr cache directory from `~/.R/lintr_cache` to `R_user_dir("lintr", "cache")` (#1062, @AshesITR)
```

Comment:
this is technically a breaking change, right?

Reply:
Not really, why would it?

Reply:
it essentially invalidates any existing cache right? unless user copies it over to the new directory

Reply:
The cache needs to be invalidated anyway due to implementation changes. We touched many default linters I think.

Reply:
makes sense. let's note that in the NEWS item then? and we can fuss more about where it comes in order later

Reply:
What should we note? "The old directory can be removed without side effects."?

Reply:
Added a sentence... feel free to merge if you agree it's useful.